ARM-wvd-templates/Update exisiting WVD host pool/README.md

```diff
@@ -0,0 +1,112 @@
+Click the button below to deploy:
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FUpdate%20existing%20WVD%20host%20pool%2FmainTemplate.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FUpdate%20existing%20WVD%20host%20pool%2FmainTemplate.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+# ARM Template to Update an existing Windows Virtual Desktop hostpool

```

Comment:
I think most of the info from this point on, except for the "Reporting issues" part, doesn't apply to AddVirtualMachinesTemplate.json so maybe just delete it for now?

Reply:
ok i'll remove most of it